app/assets/stylesheets/components/prefilled_banner.css

```
@@ -0,0 +1,11 @@
+.fr-notice__title_custom {
+    display: block;
+    color: var(--text-default-grey);
+    font-size: 1.375rem;
+    font-weight: 700;
+    line-height: 1.74rem;
+}
+
+.fr-notice__desc {
+    color: var(--text-default-grey);
+}
```

J'aurais namespace avec `prefilled_banner` justement.

```suggestion
.prefilled-banner .fr-notice__title {
    display: block;
    color: var(--text-default-grey);
    font-size: 1.375rem;
    font-weight: 700;
    line-height: 1.74rem;
}

.prefilled-banner .fr-notice__desc {
    color: var(--text-default-grey);
}
```

Et dans le html:

```html
<div class="fr-notice fr-notice--info prefilled-banner fr-my-3w fr-p-0">
```

là le `_custom` est (paradoxalement) trop générique.

---

D'accord avec loic, quand je parlais de scoper les noms, je pensais à ça : ajouter une classe.